src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -258,6 +258,8 @@
     <PlatformManifestFileEntry Include="stubs.c" IsNative="true" />
     <PlatformManifestFileEntry Include="synthetic-pthread.c" IsNative="true" />
     <!-- ICU-specific files -->
+    <PlatformManifestFileEntry Include="libc++.a" IsNative="true" />
```

Comment:
We probably don't need to add them to runtime pack. They are part of the WASI SDK (workload).

Reply:
It complains like in screen
<img width="1141" alt="libc_exception" src="https://user-images.githubusercontent.com/96171496/220877315-bf4b6ac4-5657-4075-a406-50615ebd185c.png">

Reply:
Because you copy it into that directory via `ICULibNativeFiles`

Reply:
Removed it, thanks!